packages/components/image/src/image.vue

```diff
@@ -107,49 +112,27 @@ const imageIndex = computed(() => {
   return previewIndex
 })
 
+const isManual = computed(() => {
+  if (loading?.value === 'eager') return false
```

Comment:
**Question:** loading should always be a `ref` right? Why are we adding optional chaining here?

Reply:
loading does not have a default value, it may not be defined.

Reply:
`loading` is a `Ref<'eager' | 'lazy'>` type right? So that `Ref` will always have `.value` attribute on it. That was my question, you can see the screenshot below.

![CleanShot 2022-05-30 at 19 44 13@2x](https://user-images.githubusercontent.com/15975785/170985539-602b8428-d4cb-4d85-8433-8980bfc6673d.png)

Reply:
toRefs -> Ref<BuildPropType<StringConstructor, "eager" | "lazy", unknown> | undefined> | **undefined**